src/components/teacher/teacher-group-tab.tsx

```diff
@@ -6,11 +6,17 @@ import { SixPackRightControls } from "../../clue/components/sixpack-right-contro
 import { Pager } from "../pager";
 import "./teacher-group-tab.sass";
 
+export enum DocumentViewMode {
+  LiveViewMode,
+  PublishedViewMode
```

Comment:
```suggestion
  Live,
  Published
```
since `ViewMode` is in `DocumentViewMode` it seems redundant to have it in each individual enumeration as well.